src/app/pages/data-element/data-element.component.ts

```diff
@@ -100,7 +100,8 @@ export class DataElementComponent implements OnInit {
         ([dataElementDetail, isBookmarked, profile, sourceTargetIntersections]) => {
           this.dataElement = dataElementDetail;
           this.isBookmarked = isBookmarked;
-          this.dataElementSearchResult = {
+          this.dataElementSearchResult = [];
+          this.dataElementSearchResult.push({
```

Comment:
Why not just `this.dataElementSearchResult = [{ ... }]`?

Reply:
Because in my rush I forgot arrays could be initialised like that. Fixed now.